docs/source/KGbib.bib

```diff
@@ -184,3 +184,11 @@ @article{InterHT
   journal   = {arXiv preprint arXiv:2202.04897},
   year      = {2022}
 }
+
+@inproceedings{TranS,
+  title		={TranS: Transition-based Knowledge Graph Embedding with Synthetic Relation Representation},
```

Comment:
I realize we are not very consistent with this, but I would suggest putting the whole title (or at least TranS) between double braces `{{...}}` otherwise the capitalization could get messed up when building the docs.

Reply:
Great thanks @AlCatt91, I have fixed this now.